chsdi/views/checker.py

```diff
@@ -1,18 +1,42 @@
 # -*- coding: utf-8 -*-
 
+from boto.exception import JSONResponseError
 from pyramid.view import view_config
 from pyramid.response import Response
+from pyramid.httpexceptions import HTTPInternalServerError
+from chsdi.views.files import FileView
 
 
-class Checker(object):
+class Checker(FileView):
 
     def __init__(self, request):
         self.request = request
+        self.dynamodb_table_name = 'geoadmin-file-'
```

Comment:
@procrastinatio , I am not sure about the name of the table here, shouldn't it be `geoadmin-file-storage`? Can you verify please?

Reply:
Actually it is not used